packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/UiThreadUtil.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.bridge
+
+import android.os.Handler
+import android.os.Looper
+import com.facebook.react.common.build.ReactBuildConfig
+ 
+/** Utility for interacting with the UI thread. */
+public object UiThreadUtil {
+ 
+    @Volatile private var sMainHandler: Handler? = null
```

Comment:
Could you avoid using hungarian notataiton case?

```suggestion
    @Volatile private var mainHandlerInternal: Handler? = null
```

Reply:
Done